arb_os/gasAccounting.mini

```diff
@@ -188,7 +188,7 @@ public impure func setGasAccountingParams(speedLimitPerBlock: uint, gasPoolMax:
         false
     } else {
         gasAccountingParams = struct {
-            SpeedLimitPerBlock: speedLimitPerBlock,
+            SpeedLimitPerSecond: speedLimitPerBlock,
```

Comment:
Units don't match here.  We can't set a perSecond variable to a perBlock value.  I'll push a commit to fix this.

Reply:
Fixed this.